docs/saofile.md

```diff
@@ -123,6 +123,22 @@ actions: [
 
 - __patterns__: Each entry can be a glob pattern which is supposed to matches __zero or one__ file in target directory.
 
+If you need a dynamic file path based on user answers, you can achieve it by passing a function to actions
+
+```js
+actions() {
+  const { name } = this.answers
+  return [
+    {
+      type: 'move',
+      patterns: {
+        'module.ts': `${ name }.module.ts`
+      }
+    }
+  ]
+}
+```
```

Comment:
let's write valid js in the `js` block, and format it 😄

Reply:
I'm not sure if I understand what you mean by `valid js`, I've remove the object shorthand syntax and format it

Reply:
if you mean the lost `{}`, it's copied from other code snippet in this page, should I fix that?